app/views/application/_navbar.haml

```diff
@@ -2,7 +2,7 @@
   .container
     .navbar-header
       = link_to "Meecon", root_path, class: 'navbar-brand'
-    %ul.nav.navbar-nav.navbar-right
+    %ul.nav.navbar-nav.navbar-right.navbar-font-size
```

Comment:
nice one! but consider case when you would like to add more styles to this `ul` element like `margin` and adding extra class `navbar-margin` wouldn't be a good idea. What I'm trying to say is if you could rename the css class to something related to this specific element for example `.navbar-navigation` adding more styles will be very handy :)